crates/re_space_view_spatial/src/contexts/annotation_context.rs

```diff
@@ -13,8 +12,8 @@ impl NamedViewSystem for AnnotationSceneContext {
 }
 
 impl ViewContextSystem for AnnotationSceneContext {
-    fn archetypes(&self) -> Vec<ArchetypeDefinition> {
-        vec![vec1::vec1![AnnotationContext::name()]]
+    fn all_required_components(&self) -> Vec<IntSet<ComponentName>> {
+        vec![std::iter::once(AnnotationContext::name()).collect()]
```

Comment:
Why not `AnnotationContext::required_components()`?

Reply:
+1, I think the previous way of having several archetypes no longer makes sense, this should be the same as in the ViewPartSystem

... which might eventually lead to a System base trait 🤔